docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Command Line Options/Disable PhoneHome.adoc

```diff
@@ -0,0 +1,18 @@
+= Disable PhoneHome in Payara Micro
+
+Payara Micro, like Payara Server, has the PhoneHome functionality enabled by default. It can be easily disabled in Payara Micro.
+
+[[disabling-micro-instance]]
+== Disable PhoneHome for Payara Micro
+
+When starting Payara Micro the Phone Home
+Service can be disabled with the command-line argument `--disablephonehome`:
+
+[source, shell]
+----
+java -jar payara-micro.jar --disablephonehome
+----
+
+== See Also
+
+* xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/PhoneHome.adoc[PhoneHome in Payara Server]
```

Comment:
As I suggested, the PhoneHome.adoc file should be moved to the parent directory:
```suggestion
* xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/PhoneHome.adoc[Phone Home in Payara Server]
```